### PR TITLE
Fix Users Query

### PR DESCRIPTION
**Pull request by petedejoy**
This query addresses the following issue: https://github.com/astronomer/houston-api/issues/77

I separated out our old, non-permission bound users query into two new workspaceUsers and systemUsers queries. The former is now what we use in Orbit to get a user's information and requires workspaceUuid as an arg, meaning you can't get a user's information without a workspace Id. The systemUsers query is the same as our old users query, but with an added system.users.list permission.

### CODE REVIEW

File: src/modules/users/User/Module.js [hunk 1]
```diff
@@ -46,7 +46,8 @@ const User = ({ users, menu, title }) => {
 }
 
 User.propTypes = {
-  users: PropTypes.array,
+  workspaceUser: PropTypes.object,
```
Comment (ashb) [inline src/modules/users/User/Module.js]: This module should stay "generic" and work for any user (wether it's at the Workspace, Deployment or System level) so we shouldn't use `workspace` prefix for any vars/properties.

Comment (ashb) [reply src/modules/users/User/Module.js]: To make that work maybe we need to subclass the User module in to a WorkspaceUser module.

File: src/modules/users/User/Module.js [hunk 2]
```diff
@@ -46,7 +46,8 @@ const User = ({ users, menu, title }) => {
...
+  username: PropTypes.string,
```
Comment (ashb) [inline src/modules/users/User/Module.js]: I don't _think_ this property is used anywhere is it? (It doesn't look like it from the diff anyway)

File: src/modules/users/User/Module.js (hunk 1 above)
Comment (ashb) [reply src/modules/users/User/Module.js]: (I think @samblackk was re-using this module for her new admin users screen)

**Comment by petedejoy** [comment]
Ok- this should be ready for final review cc @samblackk @schnie

**Pull request merged**